axelrod/strategies/axelrod_second.py

```diff
@@ -1661,7 +1661,7 @@ def strategy(self, opponent: Player) -> Action:
         recent_history = opponent.history[-5:]
 
         did_d = np.vectorize(lambda action: int(action == D))
-        number_defects = np.sum(did_d(recent_history))
+        number_defects = sum(did_d(recent_history))
```

Comment:
Out of curiosity: why this tweak?

Reply:
The type checker was complaining and breaking CI, my guess is that numpy started annotating more things and so it wanted the types to propagate. This was the easiest quick fix.

Reply:
Cool 👍